src/character.cpp

```diff
@@ -1949,7 +1949,8 @@ bool Character::pour_into( item &container, item &liquid )
     const long amount = container.get_remaining_capacity_for_liquid( liquid, *this, &err );
 
     if( !err.empty() ) {
-        add_msg_if_player( m_bad, err.c_str() );
+        add_msg_if_player( m_bad, string_format( "You can't pour %1$s in your %2$s: it %3$s.",
```

Comment:
Thinking of the translators: it is probably better to move the final "it " into the `err` string. In other words:
```C++
string_format( "You can't pour %1$s in your %2$s: %3$s", ..., err.c_str() );
```
And `err` would contain something like "It is full."

Currently translators have to be very careful to ensure the final sentence is properly formed. Making the content of `err` a complete sentence makes it easier to translate it independently of the prefix that is added here.

Strings like "is full" are difficult to translate without context.